stockflux-watchlist/package.json

```diff
@@ -1,39 +1,43 @@
 {
-  "name": "stockflux-watchlist",
-  "version": "0.1.0",
-  "private": true,
-  "dependencies": {
-    "classnames": "^2.2.6",
-    "isomorphic-fetch": "^2.2.1",
-    "jquery": "^3.3.1",
-    "malihu-custom-scrollbar-plugin": "^3.1.5",
-    "moment": "^2.24.0",
-    "openfin-layouts": "^1.0.0",
-    "react": "^16.8.6",
-    "react-dom": "^16.8.6",
-    "react-scripts": "^3.0.0",
-    "stockflux-components": "file:../stockflux-components",
-    "stockflux-core": "file:../stockflux-core",
-    "openfin-react-hooks": "file:../openfin-react-hooks",
-    "throat": "^4.1.0"
-  },
-  "scripts": {
-    "start": "react-scripts start",
-    "launch": "openfin --launch --config ./public/app.dev.json",
-    "build": "react-scripts build",
-    "test": "react-scripts test",
-    "eject": "react-scripts eject"
-  },
-  "eslintConfig": {
-    "extends": "react-app"
-  },
-  "browserslist": [
-    ">0.2%",
-    "not dead",
-    "not ie <= 11",
-    "not op_mini all"
-  ],
-  "devDependencies": {
-    "openfin-cli": "^2.0.8"
-  }
+    "name": "stockflux-watchlist",
+    "version": "0.1.0",
+    "private": true,
+    "dependencies": {
+        "classnames": "^2.2.6",
+        "d3-scale": "^2.1.2",
+        "d3-selection": "^1.3.2",
+        "d3fc": "^14.0.21",
+        "imports-loader": "^0.8.0",
```

Comment:
Is this a necessary dependency?

Reply:
No, it isn't, thanks! -> a03e818b1a82eea4f6e3cba19b30cd030588ee23